packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/internal/AppConfigurationImpl.kt

```diff
@@ -37,6 +37,7 @@ import io.realm.kotlin.mongodb.AppConfiguration.Companion.DEFAULT_BASE_URL
 public class AppConfigurationImpl constructor(
```

Comment:
This class now fails static analysis, but you can just `@Suppress("LongParameterList")` the warning. It is generally not a warning we are that concerned with fixing. At least not in the internal API.